source/getting_started.md

```diff
@@ -0,0 +1,523 @@
+# (getting_started)=Setting up Your Python Environment
+
+``` {contents}
+---
+depth: 2
+---
+```
+
+## Overview
+
+In this lecture, you will learn how to
+
+1.  get a Python environment up and running
+2.  execute simple Python commands
+3.  run a sample program
+4.  install the code libraries that underpin these lectures
+
+## Anaconda
+
+The [core Python package](https://www.python.org/downloads/) is easy to
+install but *not* what you should choose for these lectures.
+
+These lectures require the entire scientific programming ecosystem,
+which
+
+-   the core installation doesn\'t provide
+-   is painful to install one piece at a time.
+
+Hence the best approach for our purposes is to install a Python
+distribution that contains
+
+1.  the core Python language **and**
+2.  compatible versions of the most popular scientific libraries.
+
+The best such distribution is
+[Anaconda](https://www.anaconda.com/what-is-anaconda/).
+
+Anaconda is
+
+-   very popular
+-   cross-platform
+-   comprehensive
+-   completely unrelated to the Nicki Minaj song of the same name
+
+Anaconda also comes with a great package management system to organize
+your code libraries.
+
+**All of what follows assumes that you adopt this recommendation!**
```

Comment:
This could use a `note` or `warning` directive.  See, for example, the warning on https://myst-parser.readthedocs.io/en/latest/

Reply:
Yes, I haven't started working on features yet. :-)